docs/pages/mintlists.tsx

```diff
@@ -0,0 +1,51 @@
+import { NftokenFetcher } from "../utils/NftokenFetcher";
+import { Network } from "@glow-app/glow-client";
+import useSWR, { SWRResponse } from "swr";
+import { useGlowContext } from "@glow-app/glow-react";
+import { Solana } from "@glow-app/solana-client";
+import { NftokenTypes } from "../utils/NftokenTypes";
+import { InteractiveWell } from "../components/InteractiveWell";
+import { PageLayout } from "../components/PageLayout";
+import { useNetworkContext } from "../components/NetworkContext";
+
+export default function MintlistsPage() {
+  const { user } = useGlowContext();
+  const wallet = user?.address;
+
+  const { network } = useNetworkContext();
+
+  let { data: mintlists } = useMintlists({ wallet, network });
+  mintlists ??= [];
```

Comment:
I think we should avoid using mutable variables like this in React. This is probably fine, but mutability + React is not a good pattern. 

Instead we could do something like:

```
const { data } = useMintlists();
const mintlists = data ?? [];
```